src/PHPSemVerChecker/Comparator/Type.php

```diff
@@ -32,6 +33,15 @@ public static function get($type)
 			return '?'.static::get($type->type);
 		}
 
+		if ($type instanceof UnionType) {
```

Comment:
Can you please add a test in `tests/PHPSemVerChecker/Comparator/TypeComparatorTest.php` for this new use case? Thanks!